utils/prep_data.py

```diff
@@ -0,0 +1,84 @@
+""""
+original code by Xinqiang Ding <xqding@umich.edu>
```

Comment:
Same recommendation from before to reference the notebook or python script from this individual, if applicable.

Reply:
added! :D